frontend/src/app/features/admin/components/event-editor-dialog/event-editor-dialog.component.html

```diff
@@ -31,6 +31,15 @@ <h1>
         >
         <mat-error *ngIf="formGroup.controls.title.hasError('required')">{{t('required_field')}}</mat-error>
       </mat-form-field>
+      <mat-form-field>
+        <mat-label>{{t('type')}}</mat-label>
+        <mat-select [formControl]="formGroup.controls.type">
+          <mat-option value="Professional">{{t('professional')}}</mat-option>
+          <mat-option value="Community">{{t('community')}}</mat-option>
+          <mat-option value="Other">{{t('other')}}</mat-option>
```

Comment:
Kérlek ezeket változtasd meg olyanra, hogy ˋ*ngForˋ-al végigmész az ˋEventTypeˋ értékein, hogyha hozzáadunk egy új lehetőséget ne kelljen itt is redundánsan hozzáadni